crates/codegen/runtime/cargo/src/runtime/bindings/mod.rs

```diff
@@ -1,10 +1,14 @@
 #[path = "generated/binding_rules.rs"]
 mod binding_rules;
 
-use metaslang_graph_builder::ast;
-pub use metaslang_graph_builder::functions::Functions;
-pub use metaslang_graph_builder::{ExecutionConfig, ExecutionError, NoCancellation, Variables};
+use metaslang_bindings;
+pub use metaslang_bindings::BindingsError;
+use semver::Version;
 
 use crate::cst::KindTypes;
+pub type Bindings = metaslang_bindings::Bindings<KindTypes>;
+pub type Handle<'a> = metaslang_bindings::Handle<'a, KindTypes>;
```

Comment:
nit suggestion: grouping/separating imports and public API exports:

```suggestion
use metaslang_bindings;
use semver::Version;

use crate::cst::KindTypes;

pub type BindingsError = metaslang_bindings::BindingsError;
pub type Bindings = metaslang_bindings::Bindings<KindTypes>;
pub type Handle<'a> = metaslang_bindings::Handle<'a, KindTypes>;
```